Build storminess on master

Merging #150 seems to have made the build fail inconsistently and unexpectedly

https://travis-ci.org/redfin/react-server/builds/132744187
```
npm ERR! node v6.2.0
npm ERR! npm  v3.8.9
npm ERR! code ELIFECYCLE
npm ERR! react-server@0.2.12 prepublish: `npm run clean; gulp build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-server@0.2.12 prepublish script 'npm run clean; gulp build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-server package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run clean; gulp build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-server
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-server
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/build/redfin/react-server/packages/react-server-integration-tests/npm-debug.log
    at ChildProcess.exithandler (child_process.js:202:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:850:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
npm ERR! Linux 3.13.0-40-generic
npm ERR! argv "/home/travis/.nvm/versions/node/v6.2.0/bin/node" "/home/travis/.nvm/versions/node/v6.2.0/bin/npm" "run" "bootstrap"
npm ERR! node v6.2.0
npm ERR! npm  v3.8.9
npm ERR! code ELIFECYCLE
npm ERR! @ bootstrap: `npm i; lerna bootstrap`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @ bootstrap script 'npm i; lerna bootstrap'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the  package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm i; lerna bootstrap
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs 
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls 
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/build/redfin/react-server/npm-debug.log
The command "npm run bootstrap" failed and exited with 1 during .
Your build has been stopped.
```

https://travis-ci.org/redfin/react-server/builds/132732945
```
Errored while running npm script 'test' in 'react-server-cli'
Error: Command failed: npm run test 
(node:4128) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
npm ERR! Linux 3.13.0-40-generic
npm ERR! argv "/home/travis/.nvm/versions/node/v6.2.0/bin/node" "/home/travis/.nvm/versions/node/v6.2.0/bin/npm" "run" "test"
npm ERR! node v6.2.0
npm ERR! npm  v3.8.9
npm ERR! code ELIFECYCLE
npm ERR! react-server-cli@0.2.12 test: `gulp test`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-server-cli@0.2.12 test script 'gulp test'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-server-cli package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp test
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-server-cli
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-server-cli
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/build/redfin/react-server/packages/react-server-cli/npm-debug.log
    at ChildProcess.exithandler (child_process.js:202:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:850:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
Errored while running RunCommand.execute
Error: Command failed: npm run test 
(node:4128) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
npm ERR! Linux 3.13.0-40-generic
npm ERR! argv "/home/travis/.nvm/versions/node/v6.2.0/bin/node" "/home/travis/.nvm/versions/node/v6.2.0/bin/npm" "run" "test"
npm ERR! node v6.2.0
npm ERR! npm  v3.8.9
npm ERR! code ELIFECYCLE
npm ERR! react-server-cli@0.2.12 test: `gulp test`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the react-server-cli@0.2.12 test script 'gulp test'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-server-cli package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     gulp test
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs react-server-cli
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls react-server-cli
npm ERR! There is likely additional logging output above.
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/build/redfin/react-server/packages/react-server-cli/npm-debug.log
    at ChildProcess.exithandler (child_process.js:202:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:850:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:215:5)
npm ERR! Test failed.  See above for more details.
The command "npm test" exited with 1.
Done. Your build exited with 1.
```

--- a/packages/react-server-cli/package.json
+++ b/packages/react-server-cli/package.json
@@ -4,7 +4,7 @@
   "description": "A command line to run react-server",
   "main": "target/index.js",
   "scripts": {
-    "prepublish": "npm run clean; gulp",
+    "prepublish": "gulp",
     "test": "gulp test",
     "clean": "rimraf target npm-debug.log*"
   },

--- a/packages/react-server-data-bundle-cache/package.json
+++ b/packages/react-server-data-bundle-cache/package.json
@@ -7,7 +7,7 @@
   "author": "Bo Borgerson",
   "license": "Apache License 2.0",
   "scripts": {
-    "prepublish": "npm run clean; gulp build",
+    "prepublish": "gulp build",
     "test": "gulp test",
     "clean": "rimraf lib npm-debug.log* index.js"
   },

--- a/packages/react-server-gulp-module-tagger/package.json
+++ b/packages/react-server-gulp-module-tagger/package.json
@@ -5,8 +5,7 @@
   "main": "index.js",
   "scripts": {
     "test": "gulp test",
-    "clean": "rimraf target npm-debug.log*",
-    "prepublish": "npm run clean"
+    "clean": "rimraf target npm-debug.log*"
   },
   "repository": "redfin/react-server",
   "author": "Redfin",

--- a/packages/react-server-test-pages/package.json
+++ b/packages/react-server-test-pages/package.json
@@ -5,7 +5,7 @@
   "description": "Some pages to test out features of react-server",
   "main": "index.js",
   "scripts": {
-    "prepublish": "npm run clean; gulp build",
+    "prepublish": "gulp build",
     "start": "react-server-cli",
     "test": "gulp test",
     "debug": "node-debug --debug-brk=0 -p 9000 react-server-cli",

--- a/packages/react-server/package.json
+++ b/packages/react-server/package.json
@@ -8,7 +8,7 @@
   "author": "Navtej",
   "license": "Apache License 2.0",
   "scripts": {
-    "prepublish": "npm run clean; gulp build",
+    "prepublish": "gulp build",
     "test": "gulp test",
     "clean": "rimraf target npm-debug.log*"
   },

